5/debian/Dockerfile

```diff
@@ -123,5 +123,13 @@ VOLUME $GHOST_CONTENT
 COPY docker-entrypoint.sh /usr/local/bin
 ENTRYPOINT ["docker-entrypoint.sh"]
 
+RUN cd "$GHOST_INSTALL"; \
+	apt-get update && apt-get install -y git; \
```

Comment:
do we need a `apt-get update` here if it's already in line 58?

Reply:
Updated Dockerfile to use a different GCS adapter where git install wasn't required.